src/Governance.sol

```diff
@@ -452,34 +448,19 @@ contract Governance is Multicall, UserProxyFactory, ReentrancyGuard, Ownable, IG
         // == Rewards Conditions (votes can be zero, logic is the same) == //
 
         // By definition if _votesForInitiativeSnapshot.votes > 0 then _votesSnapshot.votes > 0
+        if (_votesForInitiativeSnapshot.votes > votingTheshold && _votesForInitiativeSnapshot.votes > _votesForInitiativeSnapshot.vetos) 
+        {
+            uint256 claim = _votesForInitiativeSnapshot.votes * boldAccrued / _votesSnapshot.votes;
```

Comment:
Nice simplification!

Reply:
Did we have a test case for what this custom precision was trying to solve? Are we sure we are not re-introducing any important rounding error?
Hopefully not thanks to the y-intercept refactor, because code looks much better now.

Reply:
Good question. Will ask Alex which test exactly is relevant here.